rebar.config

```diff
@@ -0,0 +1,3 @@
+{deps, [
+  {any, {git, "git@github.com:flitbit/any.git", {branch, "master"}}}
```

Comment:
Of course this needs to point back to shortishly/any whenever a [rebar3 compatible PR is accepted](https://github.com/shortishly/any/pull/1).